google/cloud/ndb/model.py

```diff
@@ -3836,7 +3783,9 @@ def _comparison(self, op, value):
         value = self._do_validate(value)
         filters = []
         match_keys = []
-        for prop in self._model_class._properties.values():
+        # Somehow, Python 2 and 3 return values in diiferent order, sort them.
```

Comment:
typo: different

Also, is there a production reason to sort them, or is it only to provide consistent ordering in unit tests? If the latter, I'd tend to sort in the test rather than in the code proper.